### ISSUE TITLE
xcparse doesn't support space in file names & paths

### ISSUE THREAD

**Issue by vince4** [ChargePoint/xcparse#10]
Hi,

I'm trying to use xcparse to extract screenshots from xcresult. 
It works fine when xcresult has no space in its name, but when it's not the case, xcparse return an error : 

**My command**
```
xcparse -s 'Test-Transient Testing-2019.10.02_18-10-58-+0200.xcresult' screen
```
**Result**
```
error: unexpected argument Testing-2019.10.02_18-10-58-+0200.xcresult; use --help to list available arguments
```

**Comment by abotkin-cpi** [comment]
Thanks for the report @vince4!  Yep, this seems like a failure in xcparse to properly pass the path in a way that supports spaces to xcresulttool.  I'll be addressing this in v0.2.

**Comment by abotkin-cpi** [comment]
@vince4 Run`brew upgrade xcparse` on machines that already have xcparse installed in order to update to v0.1.1 that includes the fix for this.  If xcparse is not installed, run the normal install command `brew install chargepoint/xcparse/xcparse`

**Comment by vince4** [comment]
Thank you @abotkin-cpi !
I just tested and I don't have this error any more. 

Now I have an other error : `Illegal instruction: 4` but it's an other issue. 
xcparse does'nt seems to work with `Test-Transient Testing` xcresult folders. 
I'm generating those xcresults using command : 
```
xcodebuild -xctestrun Scheme_Project_iphonesimulator13.0-x86_64.xctestrun test-without-building
```

Maybe I have made some mistakes.

**Comment by abotkin-cpi** [comment]
No, you’re not doing anything wrong.  The way we parse right now is very error prone and overly strict right now.  I’m working on a change to fix the Illegal Instruction issue.  Was hoping to get it in by Friday but may be closer to Monday.  Will update this ticket once it is in.

**Comment by abotkin-cpi** [comment]
@vince4 Changes were integrated in 0.3 that should resolve the Illegal Instruction issue you saw.  Run `brew upgrade xcparse` to get that version.  If you see it, let me know!

**Comment by vince4** [comment]
Hi @abotkin-cpi, thank you for the update. I have been away for a while and I just tried the version 1.0.0. 
Great improvements !

So I retried to run `xcparse` on `xcresult` from command : `xcodebuild -xctestrun Scheme_Project_iphonesimulator13.0-x86_64.xctestrun test-without-building` and it works so great !!!!

However I found a new issue for iPhone Xr, but it seems to come from the `xcrun xcresulttool export` command. I open a new issue for this.
